src/olympia/addons/tests/test_views.py

```diff
@@ -2128,6 +2129,7 @@ def test_all_without_and_with_unlisted_user_but_not_author(self):
             self.url, data={'filter': 'all_with_unlisted'})
         assert response.status_code == 403
 
+    @override_switch('beta-versions', active=True)
     def test_beta_version(self):
         self.old_version.files.update(status=amo.STATUS_BETA)
         self._test_url_only_contains_old_version(filter='only_beta')
```

Comment:
I don't think anyone uses that API parameter, but it should probably return a 400 if the switch is off (and if this is done, the API docs should be updated as well)

Reply:
Fixed in https://github.com/mozilla/addons-server/pull/7436/commits/21ea5185105f04f3c3f117fe6bf799967b36f1f8.

Reply:
Please also patch docs to remove any references to that filter value

Reply:
I don't believe the docs have waffle support, so the documentation will be updated as soon as the patch lands, even if the waffle hasn't been switched yet. Are you ok with that?

Reply:
Yeah that's fine. We don't want people to start relying on that parameter now anyway, so might as well stop documenting it asap.

Reply:
Alright, done in https://github.com/mozilla/addons-server/pull/7436/commits/86bd896a63dd4dd566f259dda4d15f1f6e4d3c78.